src/components/vm/disks/diskAdd.jsx

```diff
@@ -63,7 +63,22 @@ function getFilteredVolumes(vmStoragePool, disks) {
     return filteredVolumesSorted;
 }
 
-const SelectExistingVolume = ({ idPrefix, storagePoolName, existingVolumeName, onValueChanged, vmStoragePools, vmDisks }) => {
+function getDiskUsageMessage(vms, storagePool, volumeName) {
+    const isVolumeUsed = getStorageVolumesUsage(vms, storagePool);
+
+    if (!isVolumeUsed[volumeName] || (isVolumeUsed[volumeName].length === 0))
+        return null;
+
+    const vmsUsing = isVolumeUsed[volumeName].join(', ');
+    let text = _("This volume is already used by ") + vmsUsing;
```

Comment:
This doesn't work -- not every language can put the object at the very end. Please use "... by $0" and cockpit.format().